starlette/formparsers.py

```diff
@@ -268,7 +269,7 @@ async def parse(self) -> FormData:
         except MultiPartException as exc:
             # Close all the files if there was an error.
             for file in self._files_to_close_on_error:
-                file.close()
```

Comment:
this should be in an AsyncExitStack - if any aclose is cancelled we should continue to run the remaining ones